src/io/inputs.jl

```diff
@@ -121,76 +122,75 @@ end
 Load cluster-level data for a given attribute in a netCDF as a NamedDimsArray.
 """
 function load_nc_data(data_fn::String, attr::String, site_data::DataFrame)::NamedDimsArray
-    local loaded::NamedDimsArray
-    local i::Int64
+    local loaded_data::NamedDimsArray
 
-    ds = NCDataset(data_fn, "r")
-    dim_names = keys(dimsize(ds[attr]))
-    nd = ndims(ds[attr])
-
-    # Load all data (need same number of colons as number of dimensions)
-    data = ds[attr].var[[(:) for _ in 1:nd]...]
+    NetCDF.open(data_fn; mode=NC_NOWRITE) do nc_file
+        data::Array{<:AbstractFloat} = NetCDF.readvar(nc_file, attr)
+        dim_names::Vector{Symbol} = [Symbol(dim.name) for dim in nc_file.vars[attr].dim]
```

Comment:
```suggestion
        dim_names::Vector{Symbol} = Symbol[Symbol(dim.name) for dim in nc_file.vars[attr].dim]
```

Just adding a type hint for the Vector as well.